crates/trigger/src/cli/stdio.rs

```diff
@@ -86,6 +94,35 @@ impl StdioLoggingExecutorHooks {
             _ => Ok(()),
         }
     }
+
+    fn truncate_log_files<F: RuntimeFactors>(
+        &self,
+        configured_app: &spin_factors::ConfiguredApp<F>,
+    ) {
+        let sanitized_component_ids: Vec<String> = configured_app
+            .app()
+            .components()
+            .map(|c| sanitize_filename::sanitize(c.locked.id.clone()))
+            .collect();
+
+        for sanitized_component_id in sanitized_component_ids {
+            if let Some(stdout_log_path) = sanitized_log_path(
+                self.log_dir.as_deref(),
+                &sanitized_component_id,
+                STDOUT_LOG_FILE_SUFFIX,
+            ) {
+                let _ = std::fs::File::create(stdout_log_path);
```

Comment:
I think you don't need the `let` here - the `_` suffices as a discard indicator.
```suggestion
                _ = std::fs::File::create(stdout_log_path);
```

(I may be mis-remembering though)